packages/web/src/client/components/Headers/SiteHeader.stories.tsx

```diff
@@ -1,33 +1,40 @@
-import { Nav, NavProps } from './SiteHeader'
-import { Story, Meta } from '@storybook/react'
-
-const Template: Story<NavProps> = (args) => <Nav {...args} />
-
-export const WorkPage = Template.bind({})
-
-WorkPage.args = {
-  currentPath: '/projects/del-core',
-}
-
-export const ApproachPage = Template.bind({})
-
-ApproachPage.args = {
-  currentPath: '/approach',
-}
-
-export const TeamPage = Template.bind({})
-
-TeamPage.args = {
-  currentPath: '/team',
-}
-
-export const HomePage = Template.bind({})
-
-HomePage.args = {
-  currentPath: '/',
-}
+import { Nav } from './SiteHeader'
+import { ComponentMeta, ComponentStory } from '@storybook/react'
 
 export default {
-  title: 'Components/Nav',
+  title: 'Nav',
   component: Nav,
-} as Meta
+} as ComponentMeta<typeof Nav>
+
+export const NoLinks: ComponentStory<typeof Nav> = () => (
+  <Nav items={fixtures.noLinks} />
+)
+
+export const WithLinks: ComponentStory<typeof Nav> = () => (
+  <Nav items={fixtures.withLinks} />
+)
+
+const fixtures = {
+  noLinks: undefined,
+  withLinks: [
```

Comment:
I'd make this a stronger case for variation, display the behaviours, it doesnt show on the home root, it has an active state. This is missing a lot of valuable information.